src/core/js/views/notifyView.js

```diff
@@ -19,7 +19,8 @@ define([
         },
 
         setupEventListeners: function() {
-            this.listenTo(Adapt, 'remove', this.remove);
+            this.listenTo(Adapt, 'remove', this.closeNotify);
```

Comment:
Now that there's quite a few of these, would you mind changing this to the object style i.e.
```js
this.listenTo(Adapt, {
    'remove', this.closeNotify,
    'page:scrollTo': this.closeNotify,
    'device:resize': this.resetNotifySize,
    'accessibility:toggle': this.onAccessibilityToggle
});
```